src/miral/basic_window_manager.cpp

```diff
@@ -1042,7 +1042,7 @@ void miral::BasicWindowManager::place_and_size_for_state(
             return;
     }
 
-    auto const display_area = outputs.bounding_rectangle();
+    auto const display_area = const_cast<miral::BasicWindowManager*>(this)->active_output();
```

Comment:
Maybe we should add a note to make BasicWindowManager::active_output() const?

I see why it's been done, but it's a little surprising.